forge-ai/src/main/java/forge/ai/SpellApiToAi.java

```diff
@@ -167,6 +167,7 @@ public enum SpellApiToAi {
             .put(ApiType.Shuffle, ShuffleAi.class)
             .put(ApiType.SkipPhase, SkipPhaseAi.class)
             .put(ApiType.SkipTurn, SkipTurnAi.class)
+            .put(ApiType.Specialize, ChooseColorAi.class)
```

Comment:
have not yet tested it for AI play

enough bugs to work out on human side still